config/babel.prod.js

```diff
@@ -9,8 +9,7 @@
 
 module.exports = {
   presets: [
-    'babel-preset-es2015',
```

Comment:
Yeah that's it, some people would rather be compliant to the spec even though there's more output, etc